packages/use-presence/src/index.tsx

```diff
@@ -107,3 +109,57 @@ export default function usePresence(
     isExiting
   };
 }
+
+export type UseUniqueDataPresenceOptions<T> = {
```

Comment:
Maybe we should move the hooks to separate files. I'm not entirely sure to be honest, but maybe some bundlers would benefit from that in regards to tree shaking. We can still export both from an `index.tsx` file. At least from a maintainability perspective that might be handy.

Reply:
Done, now there's `use-presence.tsx`, `use-presence-switch.tsx`, and a barrel `index.tsx` 👍